appengine/src/main/java/com/google/googleinterns/gscribe/config/MySQLConfig.java

```diff
@@ -17,14 +17,11 @@
 package com.google.googleinterns.gscribe.config;
 
 public class MySQLConfig {
-    private final String url;
-    private final String userName;
-    private final String password;
+    private String url;
+    private String userName;
+    private String password;
 
-    public MySQLConfig(String url, String userName, String password) {
```

Comment:
How are the variables set? Is it getting deserialized?

Reply:
Forgot to add a new MySQLConfig file in this PR. This will be removed and a normal constructor for deserialization is used there. Will update the PR to add a new MySQLConfig file.